pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -212,73 +211,17 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 
 	proxyInstances := node.ServiceInstances
 	noneMode := node.GetInterceptionMode() == model.InterceptionNone
-	listeners := make([]*xdsapi.Listener, 0)
-
-	actualWildcard, actualLocalHostAddress := getActualWildcardAndLocalHost(node)
+	_, actualLocalHostAddress := getActualWildcardAndLocalHost(node)
 
 	builder := NewListenerBuilder(node)
 	if mesh.ProxyListenPort > 0 {
-		// Notes that the the else branch works for both split and non-split cases.
```

Comment:
Won't this break compatibility? If I upgrade to istio 1.3, my proxies (running 1.2) will get this new inbound listener config but won't have the iptables rules set up properly

Reply:
Nevermind, I think later you mentioned it would just be an extra, ignored listener for these proxies?

Reply:
Yes. its just an extra listener. and this listener will only work if iptables is setup properly.